examples/ccvm_boxqp_plot.py

```diff
@@ -58,23 +58,42 @@
     }
 
     # Customize machine time calculating function
-    def cpu_machine_func(matching_df: pd.DataFrame):
+    def cpu_machine_func(matching_df: pd.DataFrame, **_: any) -> float:
         return np.mean(matching_df["solve_time"].values)
 
-    plot_fig, plot_ax = ccvmplotlib.plot_TTS(
+    tts_plot_fig, tts_plot_ax = ccvmplotlib.plot_TTS(
         metadata_filepath=metadata_filepath,
         problem="BoxQP",
         machine_time_func=cpu_machine_func,
     )
 
-    ccvmplotlib.apply_default_tts_styling(plot_fig, plot_ax)  # apply default styling
+    ccvmplotlib.apply_default_tts_styling(
+        tts_plot_fig, tts_plot_ax
+    )  # apply default styling
     plt.show()  # show plot in a new window
 
+    def cpu_energy_max_func(matching_df: pd.DataFrame, problem_size: int) -> float:
+        machine_time = np.mean(matching_df["solve_time"].values)
+        power_max = machine_parameters["cpu_power"][problem_size]
+        energy_max = power_max * machine_time
+        return energy_max
+
+    ets_plot_fig, ets_plot_ax = ccvmplotlib.plot_ETS(
+        metadata_filepath=metadata_filepath,
+        problem="BoxQP",
+        energy_max_func=cpu_energy_max_func,
+    )
+
+    ccvmplotlib.apply_default_ets_styling(
+        ets_plot_fig, ets_plot_ax
+    )  # apply default styling
+    plt.show()
+
     # If PLOT_OUTPUT_DIR not exists, create the path
     if not os.path.isdir(PLOT_OUTPUT_DIR):
         os.makedirs(PLOT_OUTPUT_DIR)
         print("Plot folder doesn't exist yet. Creating: ", PLOT_OUTPUT_DIR)
 
     # Save to local
-    plot_fig.savefig(PLOT_OUTPUT_DEST)
+    tts_plot_fig.savefig(PLOT_OUTPUT_DEST)
     print(f"Sucessfully saved the plot to {PLOT_OUTPUT_DEST}")
```

Comment:
I think we need this line to be duplicated: one for ETS and one for TTS.
Then let's move the TTS one up with the rest of the TTS code and have the ETS one here.